src/main/java/com/rk/rentacarapp/services/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.rk.rentacarapp.services.impl;
+
+import com.rk.rentacarapp.model.User;
+import com.rk.rentacarapp.repository.UserRepository;
+import com.rk.rentacarapp.services.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class UserServiceImpl implements UserService {
+
+    @Autowired
+    UserRepository ur;
```

Comment:
This is kind of like a single letter variable, you should use it (Clean code)

userRepository is perfectly fine